src/store/actions/countriesAction.js

```diff
@@ -0,0 +1,49 @@
+import {
+  GET_COUNTRIES, SET_ERROR, SET_LOADING, SET_CURRENT, CLEAR_CURRENT,
+} from './types';
+
+// Set loading to true
+export const setLoading = () => ({
+  type: SET_LOADING,
+});
+
+export const getCountries = () => async (dispatch) => {
+  setLoading();
+  try {
+    const res = await fetch('https://restcountries.com/v2/continent/europe');
+    const data = await res.json();
+
+    dispatch({
+      type: GET_COUNTRIES,
+      payload: data,
+    });
+  } catch (error) {
+    dispatch({
+      type: SET_ERROR,
+      payload: error.ressponse.data,
```

Comment:
- [ ] I was also unable to run your code locally due to the error below caused by this line of code. I noticed you had a typo on the response with a double s but even when I correct the spelling on my computer, I still get the error. Please kindly fix this.
![image](https://user-images.githubusercontent.com/60772732/133893201-cc170a43-2e87-4c6c-9312-adb89aadb2a6.png)